mungegithub/github/github.go

```diff
@@ -726,12 +726,17 @@ func (obj *MungeObject) AddLabel(label string) error {
 func (obj *MungeObject) AddLabels(labels []string) error {
 	config := obj.config
 	prNum := *obj.Issue.Number
-	config.analytics.AddLabels.Call(config, nil)
 	glog.Infof("Adding labels %v to PR %d", labels, prNum)
 	if len(labels) == 0 {
 		glog.Info("No labels to add: quitting")
```

Comment:
Not your fault, but you are here. This is a pretty bad error message. At least say the obj.Number and maybe the obj.Issue.Labels...